### PR TITLE
Remove the hawkular-aio image and update the README

### PR DESCRIPTION
**Pull request by objectiser**

### CODE REVIEW

**Comment by objectiser** [comment]
@pavolloffay Could you review please?

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Hawkular Docker images
+# Hawkular APM Docker images
 
-This is a set of Docker images related to Hawkular:
+This is a set of Docker images related to [Hawkular APM (Application Performance Management)](https://github.com/hawkular/hawkular-apm):
 
-- [hawkular-aio](https://registry.hub.docker.com/u/jboss/hawkular-aio) Hawkular All-In-One, runs embedded Cassandra and all you need in a single container
 - [hawkular-apm-server](https://registry.hub.docker.com/u/jboss/hawkular-apm-server) Hawkular APM, runs a standalone Application Performance Management server with embedded Elasticsearch (by default)
```
Comment (pavolloffay) [inline README.md]: It is not a change in this PR but this link redirects me to my page or root url of the hub.

This link should work:
https://hub.docker.com/r/jboss/hawkular-apm-server/

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+
 - [hawkular-apm-server-dev](https://registry.hub.docker.com/u/jboss/hawkular-apm-server-dev) Hawkular APM for developers, runs a standalone Application Performance Management server with embedded Elasticsearch (by default) and a default user/password
```
Comment (pavolloffay) [inline README.md]: The same as above:

https://hub.docker.com/r/jboss/hawkular-apm-server-dev/

**Review by pavolloffay — COMMENTED** [review]
LGTM :+1: just URLs to the hub does not work.

**Comment by objectiser** [comment]
@pavolloffay Thanks!

**Pull request merged**